pkg/kubelet/cm/cpumanager/policy_static_test.go

```diff
@@ -463,7 +463,7 @@ func TestStaticPolicyAdd(t *testing.T) {
 			}
 
 			if !cset.Intersection(st.defaultCPUSet).IsEmpty() {
-				t.Errorf("StaticPolicy AddContainer() error (%v). expected cpuset %v to be disoint from the shared cpuset %v",
+				t.Errorf("StaticPolicy AddContainer() error (%v). expected cpuset %v to be disjointed from the shared cpuset %v",
```

Comment:
"disjoint", not "disjointed"
(and again below)

Reply:
Thank you, I will update the two words.